cds_ils/migrator/relations/api.py

```diff
@@ -116,25 +116,30 @@ def migrate_document_siblings_relation():
                     document_class, legacy_pid_type, relation["related_recid"]
                 )
             except PIDDoesNotExistError as e:
-                pass
-
-            # try to find sibling in series
-            if related_sibling is None:
-                try:
-                    related_sibling = get_record_by_legacy_recid(
-                        series_class,
-                        legacy_pid_type,
-                        relation["related_recid"],
+                continue
+            rec_type = related_sibling["$schema"].split("/")[-1].split("-")[0]
+            # If it's series it must be a multipart, so we search for the
+            # first volume and link it.
+            if rec_type == "series":
+                doc_search = document_search.query(
```

Comment:
I would suggest to use a `term` search instead?

Reply:
I am not sure what are you trying to solve here, let's maybe discuss in a call

Reply:
I am trying to relate the MULTIVOLUMES-X which have the legacy_recid from the series minted but not the legacy_recid from the documents, when relating them we need to search for the Multipart and return the first document of that multipart, to match the standards with that document, if I am not mistaken. But we can discuss it better in a call.